### PR TITLE
Coworking Space Reservation App HW1

### PR DESCRIPTION
**Pull request by abdyevas**
Added Main, Admin and Customer classes

### CODE REVIEW

File: Admin.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Admin {
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<HashMap<String, Object>> spaces = new ArrayList<>();
```
Comment (elnurmes) [inline Admin.java]: why we need hashmap inside list? its over engineering, you can simply use ArrayList and directly store objects )

File: Admin.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    private ArrayList<HashMap<String, Object>> reservations;
+    private int lastId = 0;
+
+    public Admin(ArrayList<HashMap<String, Object>> reservations) {
+        this.reservations = reservations;
+    }
+
+    public void adminMenu() {
+        while (true) {
+            System.out.println("-".repeat(40));
```
Comment (elnurmes) [inline Admin.java]: try to use java text blocks: https://www.baeldung.com/java-text-blocks

File: Customer.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+    public Customer(ArrayList<HashMap<String, Object>> spaces, ArrayList<HashMap<String, Object>> reservations) {
+        this.spaces = spaces;
+        this.reservations = reservations;
+    }
+
+    public void customerMenu() {
+        while (true) {
+            System.out.println("-".repeat(40));
+            System.out.println("|   Customer Menu:                     |");
```
Comment (elnurmes) [inline Customer.java]: same java blocks

**Review by elnurmes — CHANGES_REQUESTED** [review]
Hi! I left some comments, pls check and fix them)

File: Admin.java (hunk 1 above)
Comment (abdyevas) [reply Admin.java]: Thanks for the review. I have added Reservations and Spaces classes to store the data.

**Review by elnurmes — APPROVED** [review]
you can merge it

**Pull request merged**